azure-kusto-ingest/azure/kusto/ingest/_ingestion_blob_info.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, blob_descriptor, ingestion_properties, auth_context=None):
         if ingestion_properties.mapping_reference:
```

Comment:
IMHO we should choose either "mapping_reference" or "ingestionMappingReference" as both have the same use. If we are changing the name we should declare a breaking change.

Reply:
I do agree with you that the propery to be used when constructing `IngestionBlobInfo` is `ingestionMappingReference`. Thanks for that.
But, with regards to removing `mapping_reference` completely, won't a better solution be to issue a deprecation warning and to remove it a couple of versions down the road? (see latest commit for both fixes)